### PR TITLE
Add EMA

### PR DESCRIPTION
**Pull request by zbraniecki**
Fine tuned version of #8.

@ladamski, @stasm - this is what I came up with based on Lucas' PR.

Do you think it addresses #8 and #11?

Or is that not a good representation of what you wanted to show?

### CODE REVIEW

**Comment by zbraniecki** [comment]
Screenshot_2020-03-24 COVID-19 Regional Relative Timeseries

File: js/helpers.js [hunk 1]
```diff
@@ -173,3 +179,12 @@ function getType(types, selectedType) {
...
   throw new Error(`Unknown type: ${selectedType}`);
 }
+
+function calculateEMA(daysSet, idx, range, type) {
+  if (idx < 0) {
+    return daysSet[idx][type];
+  }
+
+  var k = 2/(range + 1);
+  return daysSet[idx][type] * k + daysSet[idx - 1][type] * (1 - k);
```
Comment (ladamski) [inline js/helpers.js]: I don't think this generates a valid EMA actually, EMA is an iterative function that operates on its output series.  For example `daysSet[idx - 1][type]` should reference the previous EMA output value not the source dataset.  That's why I generated the EMA dataset in a single pass

Comment (zbraniecki) [reply js/helpers.js]: I see. Makes sense. I don't mingle with data yet (I am considering doing it), so for now I'd prefer to go the "slow" route and see if the overhead is bareable.

**Comment by zbraniecki** [comment]
I made the calculation use the previous ema. Does it look better?

**Comment by zbraniecki** [comment]
Screenshot_2020-03-24 COVID-19 Regional Relative Timeseries(1)

**Comment by ladamski** [comment]
> I made the calculation use the previous ema. Does it look better?

Yup that looks great!

**Pull request merged**